packages/odyssey-react/src/components/Radio/RadioGroup/index.tsx

```diff
@@ -75,47 +71,48 @@ export interface Props
  * Radios appear as a ring shaped UI accompanied by a caption that allows
  * the user to choose only one option at a time.
  */
-const RadioGroup = forwardRef<HTMLFieldSetElement, Props>((props, ref) => {
+const RadioGroup = (props: Props) => {
```

Comment:
@jeffbelser-okta we went with your suggestion here, but the linter isn't having it:

```Missing return type on function  @typescript-eslint/explicit-module-boundary-types```

Do you have a recommendation?

Reply:
the linter is telling us something useful that its generally easier to reason about types across module boundaries if functions explicitly type their return values. In our case here you can always inspect the result of the value your function is assigned to see the inferred types, so: `JSX.Element`

all together:
```
const RadioGroup = (props: Props): JSX.Element => {
```

Reply:
I see. Thanks for clarifying, I will update this accordingly.